samples/HttpServer_FirmwareUpload/app/application.cpp

```diff
@@ -3,10 +3,120 @@
 #include <Data/Stream/LimitedMemoryStream.h>
 #include <MultipartParser/HttpMultipartResource.h>
 #include <Data/Stream/RbootOutputStream.h>
+#include <sodium/crypto_sign.h>
+#include "FirmwareVerificationKey.h"
 
 HttpServer server;
 String lastModified;
 
+class SignedRbootOutputStream : public RbootOutputStream
+{
+	const uint8_t* const verificationKey_;
```

Comment:
Please don't use underscores as prefix or suffix in a variable, function/method or class names.

Reply:
Fixed.
(FYI: Unlike leading underscores, a trailing underscore does result in a reserved symbol. This  naming convention is quite common to denote private members (e. g. boost))